contrib/tests.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+import itertools
+import subprocess
+import sys
+import os
+from pathlib import Path
+
+FEATURES = ['rand-std', 'recovery', 'endomorphism', 'lowmemory', 'serde']
+MSRV = 1.22
+RUSTC = None
+
+
+def call(command, cwd):
+    print(command)
+    try:
+        assert subprocess.check_call(command, shell=True, cwd=str(cwd)) == 0
```

Comment:
The assert is superfluous. Is it used for reader's convenience?

(https://docs.python.org/3.5/library/subprocess.html#subprocess.check_call)

Reply:
you're right. I changed the method multiple times and didn't realize this one throws and I can remove the assert.